### ISSUE TITLE
Multiclass Classification

### ISSUE THREAD

**Issue by licesonw** [licesonw/DeepCTR#1]
For implementing support for multiclass classification, we need to adjust several parts of the code:

* Add softmax activations in `PredictionLayer` class in _core.py_
* Adjust number of neurons in `PredictionLayer` class in _core.py_
* Adjust loss function (i.e. categorical crossentropy loss) when multiclass mode is chosen
